docs/create_development_environment.md

```diff
@@ -178,11 +178,11 @@ redis-cli -x TFUNCTION LOAD REPLACE < ./dist/main.js
 And run it:
 
 ```bash
-> redis-cli TFCALL foo.foo 0
+> redis-cli TFCALL myFirstLibrary.hello 0
 "3.1415926535897931"
 ```
 
-Notice that RedisGears **only supports pure JS libraries**, a library that has a native code or use some native JS API provided by the browser or by nodejs **will not work**.
+Notice that Triggers and Functions **only supports pure JS libraries**, a library that has a native code or use some native JS API provided by the browser or by nodejs **will not work**.
```

Comment:
```suggestion
Notice that Triggers and Functions **only supports pure JS libraries**, a library that either has native code or uses some native JS API provided by the browser or by Node.js **will not work**.
```

Reply:
It may be that Node.js is referred to as "nodejs" throughout this document. If that's the case, you can ignore my suggestion to replace "nodejs" with "Node.js" in this comment.